cernopendata/modules/fixtures/data/records/cms-primary-datasets.json

```diff
@@ -140,6 +140,16 @@
       "Run2010B"
     ],
     "system_details": {
+      "container_images": [
+        {
+          "name": "docker.io/cmsopendata/cmssw_4_2_8-slc5_amd64_gcc434:latest",
+          "registry": "dockerhub"
```

Comment:
FYI changed "docker" to "dockerhub" with @katilp since that's the official name of the registry. (Keeping lowercase for `cernopendata-client` lookup friendliness.)